tools/update-authors-json

```diff
@@ -93,6 +98,16 @@ def run_production() -> None:
                         'avatar': contrib.get('author').get('avatar_url'),
                         name: contrib.get('total'),
                     }
+                    if full_names.get(username):
+                        contrib_data["fullname"] = full_names[username]
+                    else:
+                        user_details = requests.get('https://api.github.com/users/'+username).json()
```

Comment:
Use space on either side of `+` here.

Reply:
Why don't we get the error in such cases?